tests/network/expose.go

```diff
@@ -141,6 +141,40 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		return tests.WaitForJobToSucceed(job, time.Duration(120)*time.Second)
 	}
 
+	executeVirtctlExposeCommand := func(ExposeArgs []string) {
```

Comment:
You could go a step further and make these helpers return `error` and assert the returned `err`s in the specs

Reply:
> for executeVirtctlExposeCommand I also wanted to address this code-smell...

I see. From what I understand, it's preferred in KubeVirt to not use Expect in helpers, so if we want to address that code-smell, const seems like a compromise. With that said, I don't like defining a const for Expect description either.
Seems like the two accepted standards - keeping Expect out of helpers and sonar cloud analyzer go against each other in this case.


> As for runJobsAgainstIps this xfail makes thing complicated because it has this condition ipOrderNum > 0 which I have to test inside the helper it self. Do you see a solution for this?

I see, don't have a better idea

Reply:
> for executeVirtctlExposeCommand I also wanted to address this code-smell

Maybe we could even mark that smell as False Positive in the Sonar Cloud UI? I think we don't really need to care about duplication of Expect descriptions